spec/rails_helper.rb

```diff
@@ -6,6 +6,7 @@
 require 'shoulda-matchers'
 require File.join(File.dirname(__FILE__), 'support/valid_attribute')
 require File.join(File.dirname(__FILE__), 'support/factory_girl')
+require File.join(File.dirname(__FILE__), 'support/helpers/authentication')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.